src/main/resources/templates/pps/bankTransferPenaltyReference.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>
+        What type of penalty do you need to pay?
+    </title>
+</head>
+<div id="pps-penalty-reference" layout:fragment="content">
+    <form id="ppspenaltyreference" th:action="@{''}" th:object="${penaltyReferences}" method="post" class="form">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+
+                <div th:replace="fragments/ppsGlobalErrors :: globalErrors"></div>
```

Comment:
Please can you try updating this fragment expression with the `~{...}` envelope to remove the warning `[layouts/chsBaseLayout] Deprecated unwrapped fragment expression`?
See https://www.thymeleaf.org/doc/articles/thymeleaf31whatsnew.html

Please can you also update the `src/main/resources/templates/layouts/baseLayout.html` to have `serviceName = 'penalty-payment-web'`?